### ISSUE TITLE
Lab 15 Task 5 Step 10 The instructions in the lab are depricated from Azure since June 2023

### ISSUE THREAD

**Issue by brentd09** [MicrosoftLearning/AZ500-AzureSecurityTechnologies#340]
# Module: NA
#### Step: 10

Description of issue
The instructions in the lab are no longer valid as the "Alert automation (classic) dropdown list" is depricated as of June 2023

Please update the instructions to support the new way of triggering a playbook.

Here is the screen shot to support the issue:

![image](https://github.com/MicrosoftLearning/AZ500-AzureSecurityTechnologies/assets/18732050/13c96954-f4cf-4d6c-b6d8-d7df9fcf7a0e)

**Comment by brentd09** [comment]
I tried to create the Analytics Rule without the classic alert automation and then went back and edited the analytics rule and I found that the deprication warning was no longer there and that I could now assign the Classic Alert Automation play book to the rule. There seems to be some inconsistencies here in the portal regarding this. 
I also found that it you try to do the task as per the deprication warning, you r first have to give permissions to Sentinel to run the playbook. I hope this helps you find a solution for this issue.
